testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -12085,3 +12084,224 @@ fn mark_miner_as_invalid_if_reorg_is_rejected() {
     }
     miners.shutdown();
 }
+
+#[test]
+#[ignore]
+/// This test checks that the miner ignore repeat block rejections.
+fn repeated_rejection() {
```

Comment:
Make sure to add this to the GitHub actions file so it runs in CI!

Reply:
This is no longer necessary. See https://github.com/stacks-network/stacks-core/blob/58b82314b96b6ff5bc269750d9d1ffb0cdda3dd1/.github/workflows/bitcoin-tests.yml#L21-L44

Reply:
https://github.com/stacks-network/stacks-core/actions/runs/13661482996/job/38194220760?pr=5877